### PR TITLE
Voight Kampff

### PR DESCRIPTION
**Pull request by sarahs328**
Do androids dream of electric sheep?

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -7,3 +7,13 @@ title: "Hello World"
 # Hello World
 
 This is a test, this is only a test
+
+# Are you a human or are you a replicant?
+## You’re in a desert walking along in the sand when all of the sudden you look down, and you see a tortoise, it’s crawling toward you. You reach down, you flip the tortoise over on its back. The tortoise lays on its back, its belly baking in the hot sun, beating its legs trying to turn itself over, but it can’t, not without your help. But you’re not helping. Why is that?
+1. What do you mean, I'm not helping?
+2. What is a tortoise?
+3. I don't know why I would flip the turtle over in the first place
+
```
Comment (adamrb) [inline index.md:16]: I think there should be another option.

File: index.md [hunk 2]
```diff
@@ -7,3 +7,13 @@ title: "Hello World"
...
+*what do you mean, I'm not helping?*
+
+*how can she not know what she is*
```
Comment (adamrb) [inline index.md:19]: What does this mean?

File: index.md [hunk 3]
```diff
@@ -7,3 +7,13 @@ title: "Hello World"
 # Hello World
 
 This is a test, this is only a test
+
+# Are you a human or are you a replicant?
+## You’re in a desert walking along in the sand when all of the sudden you look down, and you see a tortoise, it’s crawling toward you. You reach down, you flip the tortoise over on its back. The tortoise lays on its back, its belly baking in the hot sun, beating its legs trying to turn itself over, but it can’t, not without your help. But you’re not helping. Why is that?
+1. What do you mean, I'm not helping?
+2. What is a tortoise?
+3. I don't know why I would flip the turtle over in the first place
```
Comment (adamrb) [inline index.md]: With markdown, you'll typically only use "1." for all numbers, markdown will make them the right numbers automatically.

**Review by adamrb — CHANGES_REQUESTED** [review]
Please update the list before I accept this request.

File: index.md (hunk 1 above)
Comment (sarahs328) [reply index.md:16]: There are no other options. Those are the options. Pick one.

File: index.md (hunk 2 above)
Comment (sarahs328) [reply index.md:19]: If you had watched Blade Runner, you would know what this means.

File: index.md (hunk 3 above)
Comment (sarahs328) [reply index.md]: Ack, fixed

**Comment by sarahs328** [comment]
Addressed all comments.

**Pull request merged**